terraform/modules/redis_prometheus_exporter/resource.tf

```diff
@@ -0,0 +1,37 @@
+resource "cloudfoundry_service_key" "redis-key" {
+  name = "redis-key"
+  service_instance = var.redis_service_instance_id
+}
+
+locals {
+   url = "rediss://${cloudfoundry_service_key.redis-key.credentials.host}:${cloudfoundry_service_key.redis-key.credentials.port}"
+}
+
+resource "cloudfoundry_app" "redis-exporter" {
+  name         = "redis-exporter-${var.monitoring_instance_name}"
+  space        = var.monitoring_space_id
+  docker_image = "oliver006/redis_exporter:latest"
+  ports        = [9187]
+
+  routes {
+    route = cloudfoundry_route.redis_exporter.id
+  }
+  service_binding {
+    service_instance = var.redis_service_instance_id
+  }
+  environment = {
+    REDIS_EXPORTER_WEB_LISTEN_ADDRESS = "0.0.0.0:9187"
```

Comment:
Do we need to change this? Default is 9121

Reply:
set to default value of 9121